src/main/java/nextstep/ladder/domain/Lines.java

```diff
@@ -0,0 +1,42 @@
+package nextstep.ladder.domain;
+
+import nextstep.ladder.strategy.LineStrategy;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Lines {
+    private final List<Line> lines = new ArrayList<>();
+
+    public Lines() {
+        lines.addAll(Collections.emptyList());
```

Comment:
```suggestion
        this(new ArrayList<>());
```
주 생성자를 추가하고 주 생성자를 호출하는 방식으로 구현하면 어떨까?